lxd/metadata/configuration.json

```diff
@@ -2467,6 +2467,15 @@
 							"type": "bool"
 						}
 					},
+					{
```

Comment:
In the repos Makefile there is a target called `make update-metadata`. Run this one instead as it will take care of doing all the necessary changes.

Also make sure to add those changes into a separate commit like this one: https://github.com/canonical/lxd/commit/6fcb6eb548bab263b9c3956fa01321e87ed528cd